contracts/dOrgProjectFactory.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "./dOrgProject.sol";
+import "@openzeppelin/contracts/proxy/Clones.sol";
+
+/**
+ * @dev Creates an instance of `dOrgProjectFactory`.
+ */
+
+interface GnosisSafe {
+    function setup(
+        address[] calldata _owners,
+        uint256 threshold,
+        address to,
+        bytes calldata data,
+        address fallbackHandler,
+        address paymentToken,
+        uint256 payment,
+        address payable paymentReceiver
+    ) external;
+}
+
+contract dOrgProjectFactory {
+    address public immutable treasuryWallet;
+    address public immutable dOrgProjectLogic;
+    address public immutable gnosisLogic;
+
+    event ProjectCreated(address projectAddress, address gnosisSafeAddress);
+
+    constructor(address gnosisLogic_) {
+        treasuryWallet = address(0x15344EcDc2c4EDFCB092E284d93c20F0529FD8a6);
+        dOrgProjectLogic = address(new dOrgProject());
+        gnosisLogic = gnosisLogic_;
+    }
+
+    function createProject(
+        string calldata projectName,
+        address finderWallet,
+        address[] calldata owners,
+        uint256 threshold
```

Comment:
A user will ask: "What is threshold and what number should it be at?"
Consider renaming